website/client/src/components/payments/sendGiftModal.vue

```diff
@@ -0,0 +1,638 @@
+            <div class="d-flex justify-content-center align-items-middle">
+              <span class="balance-text">
+                {{ $t('yourBalance') }}</span>
+              <span
+                  class="icon-gem balance-gem-margin"
+                  v-html="icons.gemIcon"
+                  style="display: inline-block;"
+                ></span>
+              <span
+                class="balance-gems">
+                {{ maxGems }}
+              </span>
+            </div>
+          </div>
+          <div class="d-flex flex-column justify-content-center align-items-middle mt-3">
+            <button
+            v-if="fromBal"
+            class="btn btn-primary mx-auto mt-2"
+            type="submit"
+            :disabled="sendingInProgress"
+            @click="sendGift()"
+            >
+              {{ $t("send") }}
+            </button>
+          </div>
+
+          <!-- change to buying gems page -->
+          <div
+          :class="{active: selectedPage === 'buyGems'}"
+          @click="selectPage('buyGems')"
+          class="gem-state-change"
+          >
+          {{ $t('needToPurchaseGems') }}
+          </div>
+        </div>
+      </div>
+    </div>
+  </b-modal>
+</template>
+
+<style lang="scss">
+  @import '~@/assets/scss/mixins.scss';
+  #send-gift {
+    .modal-dialog {
+      max-width: 448px;
+    }
+
+    .modal-content {
+      width: 448px;
+      border-radius: 8px;
+      box-shadow: 0 14px 28px 0 rgba(26, 24, 29, 0.24), 0 10px 10px 0 rgba(26, 24, 29, 0.28);
+    }
+    .modal-body{
+      padding: 0px;
+    }
+
+    .modal-close {
+      position: absolute;
+      width: 18px;
+      height: 18px;
+      padding: 4px;
+      right: 16px;
+      top: 16px;
+      cursor: pointer;
+
+      .icon-close {
+        width: 15px;
+        height: 15px;
+
+        & ::v-deep svg path {
+          fill: #878190;
+        }
+      }
+    }
+    #subscription-form .subscribe-option {
+      background: #F9F9F9;
+    }
+}
+</style>
+<style scoped lang="scss">
+  @import '~@/assets/scss/colors.scss';
+
+  h2 {
+    color: $purple-300;
+    padding-top: 2rem;
+  }
+
+  .avatar-spacer {
+    height: 9px;
+  }
+
+  .display-name {
+    font-size: 0.875rem;
+    font-weight: bold;
+    line-height: 1.71;
+    margin: 0px 6px 0 20px;
+  }
+
+  .display-name a:hover{
+    text-decoration: none;
+  }
+
+  .user-name {
+    font-size: 0.75rem;
+    line-height: 1.33;
+    text-align: center;
+    color: $gray-100;
+    padding-bottom: 16px;
+  }
+
+  .row {
+    background-color: $gray-700;
+    margin: 0 0 0 0;
+    min-height: 32px;
+  }
+
+  .nav {
+    font-weight: bold;
+    font-size: 0.75rem;
+    min-height: 32px;
+    padding: 16px 0 0 0;
+    color: $purple-300;
+    justify-content: center;
+  }
+
+  .nav-item {
+    display: inline-block;
+    padding: 0px 8px 6px 8px;
+  }
+
+  .nav-item:hover, .nav-item.active {
+    color: $purple-300;
+    border-bottom: 2px solid #9a62ff;
```

Comment:
Interesting, this isn't one of the colors in `colors.scss` -- is it on a stylesheet in Zeplin? it looks like we use it in a handful of places... if it's not on our stylesheet, maybe check with Tressley if it ought to be; if it's on the stylesheet, let's create a color variable in `colors.scss` and employ it here.

Reply:
It's supposed to be $purple-400, was an artifact,